DataProcessing/process.py

```diff
@@ -0,0 +1,109 @@
+import requests
+import json
+import os
+import shutil
+
+from series import Series
+from series_to_download import SERIES_IDS_BY_SURVEY_NAME
+
+
+HEADERS = {'Content-type': 'application/json'}
+API_KEY = "01a6561acaec4a16bb62e92768f8753d" # "518d7f62dd3b4a2a809fe5ba9648082b"
+MAX_TIME_FRAME_YEARS = 20 # The API allows a time frame of up to 20 years
+MAX_SERIES_IDS = 50 # The API endpoint only lets you select up to 50 series at one time (not documented)
+
+
+docs_table_file = open("../docs-table.html", "w")
+docs_table_file.write("<p>The BLS database contains over 77 million series from over 60 different surveys, but not all of them are available on QuantConnect. The following sections show the integrated surveys and the series they contain.</p>")
+
+for survey_name, series_ids in SERIES_IDS_BY_SURVEY_NAME.items():
+    docs_table_file.write(f"""
```

Comment:
The generated `docs_table_file` on daily processing won't really be used, that's expected right?